lib/coverband/collectors/coverage.rb

```diff
@@ -138,9 +138,7 @@ def initialize
         if Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2.3.0')
           raise NotImplementedError, 'not supported until Ruby 2.3.0 and later'
         end
-        unless defined?(::Coverage)
```

Comment:
I can't recall exactly but I thought I added this because it was given a warning when already loaded by things like simplecov, etc

Reply:
Interesting.. think it's ok to remove for now and add back in if needed?

Reply:
yeah should be OK